src/model/Position.ts

```diff
@@ -40,6 +44,34 @@ export class Position {
     feeGrowthInside0X128: JSBI,
     feeGrowthInside1X128: JSBI
   ) {
-    // TODO
+    let liquidityNext: JSBI;
+    if (JSBI.equal(liquidityDelta, ZERO)) {
+      assert(JSBI.lessThanOrEqual(this.liquidity, ZERO), "NP");
```

Comment:
It's actually 'burn 0 liquidity'(refresh fee income) use case here and this assertion doesn't want uninitialized position to de updated. Let's do check work to see if it should be 'greaterThanOrEqual'.

Reply:
can you confirm it should be `>= 0` instead of `> 0`?

https://github.com/Uniswap/uniswap-v3-core/blob/b2c5555d696428c40c4b236069b3528b2317f3c1/contracts/libraries/Position.sol#L54

Reply:
Sorry, it should be `greaterThan`. I can see why `lessThanOrEqual` here comes.

Reply:
yup, stupid reversed logic